mllib/src/main/scala/org/apache/spark/ml/r/RWrapperUtils.scala

```diff
@@ -41,5 +41,29 @@ object RWrapperUtils extends Logging {
         s"using new name $newFeaturesName instead")
       rFormula.setFeaturesCol(newFeaturesName)
     }
+
+    if (rFormula.getForceIndexLabel && data.schema.fieldNames.contains(rFormula.getLabelCol)) {
+      val newLabelName = s"${Identifiable.randomUID(rFormula.getLabelCol)}"
+      logWarning(s"data containing ${rFormula.getLabelCol} column and we force to index label, " +
```

Comment:
Good point. I strongly support to make this INFO since users should not concern about this issue. I'll update it soon.